src/atoms/LinkBarLinkBase.js

```diff
@@ -62,15 +74,20 @@ export const LinkBarLinkBase = styled.a`
 	transition: padding .2s;
 	background: ${(props) => {
 		if (props.isActive) {
-			// Deprecated actual css color
+			if (props.activeBackground && !props.activeBackgroundColor) {
```

Comment:
Should we not log if we get `activeBackgroundColor` and `activeBackground` ?

Reply:
Nah, activeBackground has a defaultProp set, so if both are set everything is _probably_ good.